endurain/docker-compose.yml

```diff
@@ -0,0 +1,49 @@
+version: "3.7"
+
+services:
+  app_proxy:
+    environment:
+      APP_HOST: endurain_app_1
+      APP_PORT: 8080
+      PROXY_AUTH_ADD: "false"
+
+  app:
+    image: ghcr.io/joaovitoriasilva/endurain:v0.11.2@sha256:2bd4c73e564a152ac17e336c7923e36cc6a28a3be6e67664db7ba11659455033
+    # not working rootless right now
+    # user: "1000:1000"
+    environment:
+      DB_TYPE: postgres
```

Comment:
I think we can run as non-root using the `UID` and `GID` env variables:
https://docs.endurain.com/getting-started/